### ISSUE TITLE
schematics give an error

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-ngx#2835]
adding `@fundamental-ngx/core` gives error:
```
% ng add @fundamental-ngx/core
Unable to find compatible package.  Using 'latest'.
Package has unmet peer dependencies. Adding the package may not succeed.
Installing packages for tooling via npm.
Installed packages for tooling via npm.
An unhandled exception occurred: Cannot find module '@schematics/angular/utility/project'
Require stack:
- /test/delete/testtoolbar/node_modules/@fundamental-ngx/core/schematics/ng-add/index.js
- /test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/index.js
- /test/delete/testtoolbar/node_modules/@angular/cli/utilities/json-schema.js
- /test/delete/testtoolbar/node_modules/@angular/cli/models/command-runner.js
- /test/delete/testtoolbar/node_modules/@angular/cli/lib/cli/index.js
- /usr/local/lib/node_modules/@angular/cli/lib/init.js
- /usr/local/lib/node_modules/@angular/cli/bin/ng
See "/private/var/folders/1h/8mnt0hzx72x8p_c5p9216r5c0000gn/T/ng-kTb4US/angular-errors.log" for further details.
```

```
[error] Error: Cannot find module '@schematics/angular/utility/project'
Require stack:
- /test/delete/testtoolbar/node_modules/@fundamental-ngx/core/schematics/ng-add/index.js
- /test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/index.js
- /test/delete/testtoolbar/node_modules/@angular/cli/utilities/json-schema.js
- /test/delete/testtoolbar/node_modules/@angular/cli/models/command-runner.js
- /test/delete/testtoolbar/node_modules/@angular/cli/lib/cli/index.js
- /usr/local/lib/node_modules/@angular/cli/lib/init.js
- /usr/local/lib/node_modules/@angular/cli/bin/ng
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/test/delete/testtoolbar/node_modules/@fundamental-ngx/core/schematics/ng-add/index.js:9:19)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at new ExportStringRef (/test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/export-ref.js:19:25)
    at NodeModulesEngineHost._resolveReferenceString (/test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/node-module-engine-host.js:95:21)
    at NodeModulesEngineHost.createSchematicDescription (/test/delete/testtoolbar/node_modules/@angular-devkit/schematics/tools/file-system-engine-host-base.js:174:34)
    at SchematicEngine.createSchematic (/test/delete/testtoolbar/node_modules/@angular-devkit/schematics/src/engine/engine.js:220:38)
    at CollectionImpl.createSchematic (/test/delete/testtoolbar/node_modules/@angular-devkit/schematics/src/engine/engine.js:70:29)
```

@angular/cli version
```
Angular CLI: 10.0.2
Node: 12.16.3
OS: darwin x64

Angular: 10.0.3
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.1000.2
@angular-devkit/build-angular     0.1000.2
@angular-devkit/build-optimizer   0.1000.2
@angular-devkit/build-webpack     0.1000.2
@angular-devkit/core              10.0.2
@angular-devkit/schematics        10.0.2
@angular/cli                      10.0.2
@ngtools/webpack                  10.0.2
@schematics/angular               10.0.2
@schematics/update                0.1000.2
rxjs                              6.5.5
typescript                        3.9.6
webpack                           4.43.0
```

It works with `9.1.11`

**Comment by mikerodonnell89** [comment]
unable to reproduce with a clean angular project generated using ng new

**Comment by mikerodonnell89** [comment]
This could possibly be fixed by: https://github.com/SAP/fundamental-ngx/pull/2884

**Comment by droshev** [comment]
I was thinking that we should fix this independently from the migration to Angular 10 and downport it.

**Comment by JKMarkowski** [comment]
Now schematics work with Angular 10.

**Comment by kavya-b** [comment]
Hi @JKMarkowski , when upgrading the consuming application to latest Ang10 and then calling `ng add @fundamental-ngx/core` again calls the same error:
<img width="845" alt="Screenshot 2020-07-28 at 9 55 04 AM" src="https://user-images.githubusercontent.com/53509521/88620886-b823bb80-d0bc-11ea-965f-eef5b43fd1e5.png">

If the consuming application was on Ang8.2 and we call `ng add @fundamental-ngx/core`, it works okay. But if we upgrade this application to Ang9, then to Ang10, or directly from Ang8 to Ang10 (https://update.angular.io/#9.0:10.0), and then call `ng add @fundamental-ngx/core`, it throws the above error.

**Comment by JKMarkowski** [comment]
Hi @kavya-b It's because `core@0.20` doesn't have angular 10 support. It will be included in 0.21.0
